CSS/navbar.css

```diff
@@ -0,0 +1,36 @@
+.nav {
+  background-color: rgba(5, 27, 58, 0.7);
+  box-shadow: none;
+}
+
+.nav-icon {
+  color: white;
+}
+
+.nav-icon:hover {
+  color: white;
+}
+
+.nav-dropdown {
+  display: none;
+  top: 2rem;
+  right: -1.5rem;
+  width: 8rem;
+  padding: 0.5rem;
```

Comment:
Should pixels when value is too small

Reply:
thanks saurabh, i'll keep in mind from next time